.husky/commit-msg

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Skip commit-msg hook on CI
+[ -n "$CI" ] && exit 0
+
+npx --no-install commitlint --edit $1
```

Comment:
Since `commitlint` is already installed, the `--no-install` option isn't necessary

```suggestion
npx commitlint --edit $1
```